docs/_layout.scss

```diff
@@ -0,0 +1,93 @@
+body.hxVertical {
```

Comment:
migrated from `docs.less` and modified to use full-width header layout (see SURF-1730)